src/test/java/org/ays/auth/controller/AysUserControllerTest.java

```diff
@@ -638,4 +641,94 @@ void givenValidIdAndInvalidUserUpdateRequest_whenRoleIdIsNotValid_thenReturnVali
                 .update(Mockito.anyString(), Mockito.any(AysUserUpdateRequest.class));
     }
 
+    @Test
+    void givenValidId_whenActivateUser_thenReturnSuccess() throws Exception {
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
```

Comment:
Testlerde verileri daha net görebilmek adına aşağıdaki şekilde kullanabiliriz, diğer testlerdeki kullanımlar da yavaş yavaş düzeltilecek
```suggestion
        String mockId = "793fcc5d-31cc-4704-9f0a-627ac7da517d";
```